configs/sx8m_config

```diff
@@ -76,9 +76,7 @@ CONFIG_GENERIC_IRQ_CHIP=y
 CONFIG_IRQ_DOMAIN=y
 CONFIG_IRQ_DOMAIN_HIERARCHY=y
 CONFIG_GENERIC_MSI_IRQ=y
-CONFIG_GENERIC_MSI_IRQ_DOMAIN=y
-CONFIG_HANDLE_DOMAIN_IRQ=y
-# CONFIG_IRQ_DOMAIN_DEBUG is not set
+CONFIG_GENERIC_proptietary_DEBUG is not set
```

Comment:
Were these changes intentional in the last commit? I don't know what these options do